AK/BigIntBase.h

```diff
@@ -277,6 +285,12 @@ ALWAYS_INLINE constexpr WordType div_mod_words(WordType dividend0, WordType divi
     return static_cast<WordType>(dividend / divisor);
 }
 
+template<typename WordType>
+constexpr DoubleWord<WordType> dword(WordType word0, WordType word1)
```

Comment:
why is this `word0, word1` when the rest of the functions above are `word1, word2`?

Reply:
Made it more explicit with `low` and `high` in both cases